Easydict/Feature/EventMonitor/EZEventMonitor.m

```diff
@@ -143,8 +143,15 @@ - (EZTriggerType)getAppSelectTextActionType:(NSString *)appBundleID
     wechat.appBundleID = @"com.tencent.xinWeChat";
     wechat.triggerType = EZTriggerTypeDoubleClick | EZTriggerTypeTripleClick;
     
+    // https://github.com/tisfeng/Easydict/issues/346
+    // Add Easydict self as block app
+    EZAppModel *selfApp = [[EZAppModel alloc] init];
+    selfApp.appBundleID = [[NSBundle mainBundle] bundleIdentifier];
+    selfApp.triggerType = EZTriggerTypeNone;
```

Comment:
No, that's not a good solution, and although we're not currently handling the mouse event inside Easydict, it may be used later, such as in a future main window, so we can't just turn it off here.

If there is no better solution, you can use the old workaround https://github.com/tisfeng/Easydict/commit/0b8f9e9b0b5932ea95ce91258dc114243011b82c for now.

Reply:
ok, I will use the old workaround for now. I'll see if I can find other solutions.